client/hotrod-client/src/main/java/org/infinispan/client/hotrod/ProtocolVersion.java

```diff
@@ -35,10 +36,11 @@
    PROTOCOL_VERSION_28(2, 8, new Codec28()),
    PROTOCOL_VERSION_29(2, 9, new Codec29()),
    PROTOCOL_VERSION_30(3, 0, new Codec30()),
+   PROTOCOL_VERSION_31(3, 1, new Codec31()),
```

Comment:
aren't we giving up on protocol versions? I thought we no longer increment versions and just add new operations (and keep the old ones unchanged)

Reply:
Oh, I wasn't aware. Unfortunately, this changes an existing operation by adding a new field. So I am not sure how that works. Do we have versioned operations then?

Reply:
I hope I'm not mistaken... 
just create a new operation: https://github.com/infinispan/infinispan/blob/master/server/hotrod/src/main/resources/hotrod.gr#L324

Reply:
I think Will's approach is fine for this.

Reply:
I had originally commented that I was fine with this, but the more I think about it the more I believe this deserves its own operation unless we are going to have bloom filters for listeners in general

Reply:
Sure, I can do that. I had debated about a different operation since we don't need the filter, converter, or include state.

Reply:
Ok, please do that then.